manifest_dev.yml

```diff
@@ -9,12 +9,16 @@ applications:
     services:
       - database  # aws-rds medium-psql
       - config    # user-provided service w/ DJANGO_SECRET_KEY and NEW_RELIC_LICENSE_KEY
+      - storage-s3  # S3 basic service
```

Comment:
Yup. I also have to actually create the S3 service for production, which I've so far held off on doing (until things work in dev, basically).